zulipterminal/core.py

```diff
@@ -659,6 +667,8 @@ def _raise_exception(self, *args: Any, **kwargs: Any) -> Literal[True]:
             else:
                 import traceback
 
+                full_traceback = "".join(traceback.format_exception(*exc))
```

Comment:
I'm not entirely sure how to trigger an actual exception, but based on my own test exception. I believe `format_exception` would be the better option since it provides more information (type of the exception, exception message, full traceback)

format_exception
```python
Traceback (most recent call last):
  File "/home/ubuntu/zulip-terminal/zulipterminal/ui_tools/boxes.py", line 827, in keypress
    raise ValueError("This is a test exception")
ValueError: This is a test exception
```


format_tb
```python
  File "/home/ubuntu/zulip-terminal/zulipterminal/ui_tools/boxes.py", line 827, in keypress
    raise ValueError("This is a test exception")
```

Reply:
I took a quick look at the docs again, but let's leave it for now, and we can follow-up later if it doesn't provide enough information.